### PR TITLE
Finishing off the documentation updates for cert-manager 1.16.0

### PR DESCRIPTION
**Pull request by wallrj**
Preview: https://deploy-preview-1576--cert-manager.netlify.app/docs/releases/release-notes/release-notes-1.16/

* Added contributors
* Added upgrade notes
* Generated API and CLI docs
* Updated install version variable

xref:
 * https://github.com/cert-manager/cert-manager/issues/7326

### CODE REVIEW

File: scripts/gendocs/postprocess/api-doc-postprocess.js
Comment (wallrj) [inline scripts/gendocs/postprocess/api-doc-postprocess.js:1]: `prettier.format` now seems to return a promise:
 * https://prettier-doc.netlify.app/docs/en/api#prettierformatsource--options

File: content/docs/cli/cainjector.md
Comment (wallrj) [inline content/docs/cli/cainjector.md:1]: Diff best viewed with the "Hide whitespace" option.

File: content/docs/reference/api-docs.md [hunk 1]
```diff
@@ -922,6 +935,132 @@ description: >-
...
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <code>seLinuxOptions</code>
+        <br />
+        <em>
+          <a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#selinuxoptions-v1-core">Kubernetes core/v1.SELinuxOptions</a>
```
Comment (wallrj) [inline content/docs/reference/api-docs.md:954]: We link to Kubernetes 1.19 API docs.
I considered changing it here, but it results in a huge diff, 
so I will update that in a separate PR after the release.

File: content/docs/reference/api-docs.md [hunk 2]
```diff
@@ -1826,7 +1991,12 @@ description: >-
...
+        <p>If you omit this region field, cert-manager will use the region from AWS_REGION and AWS_DEFAULT_REGION environment variables, if they are set in the cert-manager controller Pod.</p>
+        <p> The <code>region</code> field is not needed if you use <a href="https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html">IAM Roles for Service Accounts (IRSA)</a>. Instead an AWS_REGION environment variable is added to the cert-manager controller Pod by: <a href="https://github.com/aws/amazon-eks-pod-identity-webhook">Amazon EKS Pod Identity Webhook</a>. In this case this <code>region</code> field value is ignored. </p>
+        <p> The <code>region</code> field is not needed if you use <a href="https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html">EKS Pod Identities</a>. Instead an AWS_REGION environment variable is added to the cert-manager controller Pod by: <a href="https://github.com/aws/eks-pod-identity-agent">Amazon EKS Pod Identity Agent</a>, In this case this <code>region</code> field value is ignored. </p>
```
Comment (wallrj) [inline content/docs/reference/api-docs.md:1999]: * https://deploy-preview-1576--cert-manager.netlify.app/docs/reference/api-docs/#acme.cert-manager.io/v1.ACMEIssuerDNS01ProviderRoute53

File: content/docs/reference/api-docs.md [hunk 3]
```diff
@@ -4336,7 +4542,23 @@ description: >-
...
+        <br />
+        <em>int32</em>
+      </td>
+      <td>
+        <em>(Optional)</em>
+        <p> <code>renewBeforePercentage</code> is like <code>renewBefore</code>, except it is a relative percentage rather than an absolute duration. For example, if a certificate is valid for 60 minutes, and <code>renewBeforePercentage=25</code>, cert-manager will begin to attempt to renew the certificate 45 minutes after it was issued (i.e. when there are 15 minutes (25%) remaining until the certificate is no longer valid). </p>
+        <p>NOTE: The actual lifetime of the issued certificate is used to determine the renewal time. If an issuer returns a certificate with a different lifetime than the one requested, cert-manager will use the lifetime of the issued certificate.</p>
+        <p>
+          Value must be an integer in the range (0,100). The minimum effective
+          <code>renewBefore</code> derived from the <code>renewBeforePercentage</code> and <code>duration</code> fields is 5 minutes. Cannot be set if the <code>renewBefore</code> field is set.
```
Comment (wallrj) [inline content/docs/reference/api-docs.md:4560]: * https://deploy-preview-1576--cert-manager.netlify.app/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec

File: content/docs/reference/api-docs.md [hunk 4]
```diff
@@ -6783,9 +7079,29 @@ description: >-
...
+      <td>
+        <p>metricsTLSConfig is used to configure the metrics server TLS settings.</p>
+      </td>
+    </tr>
   </tbody>
 </table>
 <hr />
 <p>
-  <em> Generated with <code>gen-crd-api-reference-docs</code> on git commit <code>35e27b7</code>. </em>
+  <em> Generated with <code>gen-crd-api-reference-docs</code> on git commit <code>67c897d</code>. </em>
```
Comment (wallrj) [inline content/docs/reference/api-docs.md:7106]: * https://github.com/cert-manager/cert-manager/commit/67c897d068cf4602a62370f4152eeccfdd108b61

**Comment by inteon** [comment]
LGTM, lots of generated changes + I agree with the changes to the release notes

/approve
/lgtm

**Pull request merged**